### PR TITLE
KAFKA-6945: Add docs about KIP-373

### PR DESCRIPTION
**Pull request by viktorsomogyi**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by viktorsomogyi** [comment]
@omkreddy is there anything else you had in mind for the docs?

File: docs/security.html [hunk 1]
```diff
@@ -1266,6 +1264,13 @@ <h4 class="anchor-heading"><a id="security_authz_cli" class="anchor-link"></a><a
             <td></td>
             <td>ResourcePattern</td>
         </tr>
+        <tr>
```
Comment (omkreddy) [inline docs/security.html:1267]: Can we also update https://kafka.apache.org/documentation/#security_authz_primitives with newly Operations, Resource and also update [table the here](https://kafka.apache.org/documentation/#operations_resources_and_protocols)

Comment (viktorsomogyi) [reply docs/security.html:1267]: Done.

**Review by omkreddy — APPROVED** [review]
@viktorsomogyi Thanks for the PR. LGTM

**Comment by viktorsomogyi** [comment]
@omkreddy thank you for merging this!

**Pull request merged**